src/v2/view-builder/views/custom-app/ChallengeCustomAppResendPushView.js

```diff
@@ -13,6 +17,25 @@ const Body = BaseForm.extend(Object.assign(
     save() {
       return loc('oie.custom_app.push.resend', 'login');
     },
+
+    showMessages() {
+      let options = {};
+       if (this.options.appState.containsMessageWithI18nKey(CUSTOM_APP_UV_ENABLE_BIOMETRIC_SERVER_KEY)) {
+        // add a title for Custom App enable biometrics message during verification
+        options.content = null;
+        options.className = 'okta-verify-uv-callout-content';
```

Comment:
re-using SCSS class name as okta-verify since Custom App has the same styling as Okta Verify push